{"name": "test_404_error", "status": "passed", "description": "Тест обработки ошибки 404", "attachments": [{"name": "stdout", "source": "d648e283-f976-4a38-9607-b206da55f6c0-attachment.txt", "type": "text/plain"}], "start": 1760599069114, "stop": 1760599070141, "uuid": "38720fb5-d63a-42ca-9c37-27c62d062ca9", "historyId": "8c5edd45f53b33f4e3adabdf11981ca0", "testCaseId": "8c5edd45f53b33f4e3adabdf11981ca0", "fullName": "tests.test_api.TestJSONPlaceholderAPI#test_404_error", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestJSONPlaceholderAPI"}, {"name": "host", "value": "Dimon"}, {"name": "thread", "value": "11600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}], "titlePath": ["tests", "test_api.py", "TestJSONPlaceholderAPI"]}